File: 128-longest-consecutive-sequence/longest-consecutive-sequence.java
class Solution {
    public int longestConsecutive(int[] nums) {
        int n = nums.length;
        if(n==0) return 0;
        Arrays.sort(nums);
        int longest = 1, last = Integer.MIN_VALUE, cnt=0;
        for(int i=0;i<n;i++){
            if(nums[i]-1 == last){
                cnt++;
                last = nums[i];
            }
            if(nums[i] != last){
                cnt = 1;
                last = nums[i];
            }
            longest = Math.max(longest, cnt);
        }
        return longest;
    }
}

File: 128-longest-consecutive-sequence/longest-consecutive-sequence.py
class Solution(object):
    def longestConsecutive(self, nums):
        n = set(nums)
        longest = 0
        for i in n:
            if i-1 not in n:
                curr = i
                cnt = 1
                while curr+1 in n:
                    curr += 1
                    cnt += 1
                longest = max(longest, cnt)
        return longest
        """
        :type nums: List[int]
        :rtype: int
        """
        